R/get_it_extract_paths.R

```diff
@@ -13,16 +13,16 @@
 get_it_ltc_path <- function(it_reference = NULL, ...) {
   if (is.null(it_reference)) {
     it_ltc_path <- get_file_path(
-      directory = fs::path(get_slf_dir(), "IT_extracts"),
-      file_name_regexp = "SCTASK[0-9]{7}_LTCs\\.csv(?:\\.gz)?",
+      directory = fs::path(get_slf_dir(), "IT_extracts/anon-chi-IT"),
+      file_name_regexp = "anon-SCTASK[0-9]{7}_LTCs\\.parquet",
       ...
     )
   } else {
     it_reference <- check_it_reference(it_reference)
 
     it_ltc_path <- get_file_path(
-      directory = fs::path(get_slf_dir(), "IT_extracts"),
-      file_name = stringr::str_glue("SCTASK{it_reference}_LTCs.csv.gz")
+      directory = fs::path(get_slf_dir(), "IT_extracts/anon-chi-IT"),
+      file_name = stringr::str_glue("anon-SCTASK{it_reference}_LTCs.parquet")
     )
   }
```

Comment:
This looks all good! I think once this branch has merged into the anon_chi branch, we need to do a check for any outstanding places in other scripts using chi or `slfhelper::get_chi/anonchi`